batch_perf/results.txt

```diff
@@ -0,0 +1,35 @@
+On a 8 core 16MB - Apple MBP El Capitan
+
+                                                                             user     system      total        real
+redis input batch: 1000, lrange, ltrim, size: 5000, slow: 0ms            1.700000   0.070000   1.770000 (  0.362000)
```

Comment:
batch of 1000: Now that we have a unbounded queue between the inputs and filters - I wanted to see whether a very large batch made a significant difference.

Number of items in redis: this will only change how long the bm runs for.